### DISCUSSION TITLE
What feature would you like to see next?

### DISCUSSION THREAD

**Discussion by dbalsom** [dbalsom/marty#106] [Polls]
Help me prioritize new feature development.

**Comment by Foxlum** [comment]
I feel like the Tandy 3 Sound (SN76489/SN76496) is the most usable for an 8088 class machine, at least for sound output. PC Speaker is nice, but Tandy sound was a very popular alternative, especially for games of the era.

**Comment by dbalsom** [comment]
Hercules was a fairly simple extension to the existing MDA, so it was reasonable to add it quickly. VGA is going to be a bit trickier.

If you voted for Hercules, feel free to vote for something else :)

![image](https://github.com/dbalsom/martypc/assets/7229541/50e116e9-896d-4fc0-be4f-cd9e222eb497)

**Comment by dbalsom** [comment]
Y'all wanted some VGA?
![image](https://github.com/dbalsom/martypc/assets/7229541/ed64e23b-b3f2-490f-9415-3bb6a083e7a1)
